src/Sitemap.php

```diff
@@ -20,6 +22,17 @@ public static function create(): static
         return new static();
     }
 
+    public function addQuery(Builder $query, int $chunkSize = 1000): static
```

Comment:
In the readme you show that you can pass a query to `add`.

How is `addQuery` being called from `add`?

Reply:
My mistake, a typo, will update right now. 👍🏻

Reply:
Maybe the best way forward is to to let `add` accept a `Builder` and if it detects that, let it call `addQuery`.